extension/scripts/background.js

```diff
@@ -185,32 +185,43 @@ document.addEventListener('DOMContentLoaded', () => {
     }
   });
 
-  function _createNotification(notificationId, title, message) {
+  function _createNotification(notificationId, title, message = "{}") {
     chrome.notifications.create(notificationId, {
       type: 'basic',
       iconUrl: 'assets/images/icon-lg.png',
       title: title,
       message: message,
     });
   }
+  
+  function createNotificationsAsNecessary(oldValue, newValue) {
+    if (shouldIgnoreChanges(oldValue, newValue)) return;
+    createNewStatusNotificationifNecessary(oldValue, newValue);
```

Comment:
should this be, `createNewStatusNotificationIfNecessary` (capitalized the i)

Reply:
Thanks! Updated